.travis.yml

```
@@ -34,168 +34,202 @@ jobs:
 
     # Filebeat
     - os: linux
+      before_install: if bash dev-tools/travis_skip_changes.sh $TRAVIS_COMMIT_RANGE libbeat filebeat vendor; then echo "NOT testing"; travis_terminate 0; fi
```

Wdyt about including the reading of `$TRAVIS_COMMIT_RANGE` and the call to `travis_terminate` into the script itself, so here we only have something like this:
```suggestion
      before_install: dev-tools/travis_skip_changes.sh libbeat filebeat vendor
```
This way it would be easier to see what directories are being checked.

---

Sure, can do. I was thinking to keep it a little more 'generic', so we could make it reusable with Jenkins.

---

Seems like travis_terminate is a shell function only and can't be found from within the script. We will need to call it from before_install.

---

Oops, ok.